configure.ac

```diff
@@ -763,6 +763,7 @@ Python API: ${enable_python_libs}
 Java API: ${enable_java_libs}
 Enable HTTP Server: ${have_microhttpd}
 RDM Responder Tests: ${enable_rdm_tests}
+Ja Rule: ${BUILD_JA_RULE}
```

Comment:
I don't think this actually works currently. If we want it to show the final state (including libusb status), I may need to have a rethink.